services/frontend-v3/src/components/profileForms/personalGrowthForm/PersonalGrowthFormView.jsx

```diff
@@ -448,6 +455,19 @@ const PersonalGrowthFormView = (props) => {
           <Divider style={styles.headerDiv} />
           <Title level={3} style={styles.formTitle}>
             <FormattedMessage id="setup.talent.management" />
+            <Popover
+              content={
+                <div>
+                  <FormattedMessage id="profile.talent.management.tooltip" />
```

Comment:
i ran the code and you need a space between the message and the link for formatting reasons.

Reply:
whoops sorry thought I added that! (done now)